### ISSUE TITLE
how to pass some parameter in select2_from_ajax 

### ISSUE THREAD

**Issue by developerpratika** [Laravel-Backpack/CRUD#609]
Backpack is best Package,
i have on big issue how to pass some parameter in select2_from_ajax 
**is it possible ?**  
my case:
let i am inserting City. may table has following fields:  protected` $fillable = ['name', 'country_id','state_id','status'];`
i want display state on change of country_id only select state of select country 
is it possible ?

**Comment by iMokhles** [comment]
hi @developerpratika can you rephrase your issue please.
thanks

**Comment by adbmdp** [comment]
I think you want kind of a dependent dropdown right?
select2_from_ajax content changing depending on your first selection?

**Comment by developerpratika** [comment]
@adbmdp you are right

**Comment by tabacitu** [comment]
Hi @developerpratika ,

In that case, there's no pre-built solution for you. I recommend you [create a custom field type](https://laravel-backpack.readme.io/docs/crud-fields#section-custom-field-types) starting from the select2_from_ajax, and place some javascript there that sends the value of your first selection through the AJAX call.

You can then filter using that variable inside the CrudController method.

Hope it helps. Cheers!

**Comment by developerpratika** [comment]
ok thanks for your support . i did  same it working great :)

**Comment by devParry** [comment]
Hi @developerpratika 

I am new in **BackPack** and i have same requirement e.g ( Firstly user select any country from First select option >> on the behalf of selected country  id, I have to display all states of that  country in another Select ( drop-down list )  Could you please help me to sort out this. by sharing some step or dummy code for this. 

Thanks in  Adv.
